<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.13.1 on linux)" generated="2025-03-16T18:01:25.376503" rpa="false" schemaversion="5">
<suite id="s1" name="Monitor" source="/home/runner/workspace/monitor.robot">
<test id="s1-t1" name="Monitor Netflix Devices" line="16">
<kw name="Open Netflix">
<kw name="Open Browser" owner="SeleniumLibrary">
<msg time="2025-03-16T18:01:26.477251" level="INFO">Opening browser 'chrome' to base url 'https://www.netflix.com/YourAccount'.</msg>
<msg time="2025-03-16T18:01:46.732589" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<msg time="2025-03-16T18:01:46.732883" level="FAIL">WebDriverException: Message: Service /home/runner/.cache/selenium/chromedriver/linux64/134.0.6998.88/chromedriver unexpectedly exited. Status code was: 127
</msg>
<arg>${NETFLIX_URL}</arg>
<arg>browser=chrome</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<status status="FAIL" start="2025-03-16T18:01:26.476861" elapsed="20.260616">WebDriverException: Message: Service /home/runner/.cache/selenium/chromedriver/linux64/134.0.6998.88/chromedriver unexpectedly exited. Status code was: 127
</status>
</kw>
<kw name="Maximize Browser Window" owner="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="NOT RUN" start="2025-03-16T18:01:46.737745" elapsed="0.000041"/>
</kw>
<status status="FAIL" start="2025-03-16T18:01:26.475827" elapsed="20.262120">WebDriverException: Message: Service /home/runner/.cache/selenium/chromedriver/linux64/134.0.6998.88/chromedriver unexpectedly exited. Status code was: 127
</status>
</kw>
<kw name="Login To Netflix">
<status status="NOT RUN" start="2025-03-16T18:01:46.738324" elapsed="0.000049"/>
</kw>
<kw name="Get Active Devices">
<status status="NOT RUN" start="2025-03-16T18:01:46.738644" elapsed="0.000044"/>
</kw>
<kw name="Validate Streaming Limits">
<status status="NOT RUN" start="2025-03-16T18:01:46.738887" elapsed="0.000034"/>
</kw>
<kw name="Logout Unwanted Devices">
<status status="NOT RUN" start="2025-03-16T18:01:46.739142" elapsed="0.000031"/>
</kw>
<kw name="Close Browser">
<status status="NOT RUN" start="2025-03-16T18:01:46.739339" elapsed="0.000036"/>
</kw>
<status status="FAIL" start="2025-03-16T18:01:26.475197" elapsed="20.264362">WebDriverException: Message: Service /home/runner/.cache/selenium/chromedriver/linux64/134.0.6998.88/chromedriver unexpectedly exited. Status code was: 127
</status>
</test>
<status status="FAIL" start="2025-03-16T18:01:25.377567" elapsed="21.362692"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Monitor" id="s1" pass="0" fail="1" skip="0">Monitor</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
